### PR TITLE
The dropdown value of old JASP files have sometimes not exactly the right value

### PR DESCRIPTION
**Pull request by boutinb**
Fixes https://github.com/jasp-stats/jasp-test-release/issues/1836

Due to old bugs, the value set in a dropdown might be set with the label and not with its value. This could be solved by adding entries in the upgrade.qml, but this can be quite cumbersome, since the problem can have occurred in many older JASP versions.
We can also argue, that if a value in a JASP file is not found in the possible value list of a DropDown, then before throwing an error message, we can try to find the value in the labels of the DropDown.

### CODE REVIEW

File: Resources/Data Sets/Data Library/3. ANOVA/Alcohol Attitudes copy/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Manifest-Version: 1.0
```
Comment (JorisGoosen) [inline Resources/Data Sets/Data Library/3. ANOVA/Alcohol Attitudes copy/META-INF/MANIFEST.MF]: Why is this file here? Accidentally committed?

**Review by JorisGoosen — CHANGES_REQUESTED** [review]
Remove the Alcohol Attittudes copy...

**Comment by boutinb** [comment]
> Remove the Alcohol Attittudes copy...

Oeps... weggehaald

**Review by JorisGoosen — APPROVED** [review]
